### ISSUE TITLE
Python Client fails to push dumped data

### ISSUE THREAD

**Issue by Sparika** [skydive-project/skydive#2118]
**Summary:**
I'm trying to add a dumped topology to another analyzer using the python client, as described here http://skydive.network/documentation/api-python#dump--inject where the dumped analyzer is another analyzer with actual data.

**Steps to reproduce:**
```
 curl -o skydive.json http://10.35.8.37:30555/api/topology
```
``` 
skydive-ws-client --analyzer localhost:8082 --username admin --password password add skydive.json
```

**Expected result:**
The dumped topology should appear within the localhost analyzer, similarly to the use of the `skydive client` command.

**Actual result:**
The `skydive-ws-client` manages to read the file and send WS messages logging messages such as 
```
DEBUG:skydive.websocket.client:Sending message: {"UUID": "3871054e67954938866adb239eb5aea8", "Namespace": "Graph", "Type": "NodeAdded"
...
``` 
However, nothing appears on the analyzer window and requests for topology continue to return empty results. Here is the debug log of the analyzer: 
```
2019-11-25T13:25:49.010Z        INFO    server/server.go:338 (*Server).serveLogin       f31c687047b5: User admin authenticated with noauth backend with roles [admin]
2019-11-25T13:25:49.014Z        DEBUG   websocket/server.go:68 (*Server).serveMessages  f31c687047b5: Enforcing websocket for /ws/publisher, admin
2019-11-25T13:25:49.014Z        DEBUG   config/rbac.go:49 (*logger).Print       f31c687047b5: Request: admin, websocket, /ws/publisher ---> true
2019-11-25T13:25:49.014Z        DEBUG   websocket/server.go:80 (*Server).serveMessages  f31c687047b5: Serving messages for client Test for pool /ws/publisher type : [*websocket.Pool]
2019-11-25T13:25:49.014Z        INFO    websocket/server.go:118 (*Server).newIncomingClient     f31c687047b5: New WebSocket Connection from 172.17.0.1:53296 : URI path /ws/publisher
2019-11-25T13:25:49.015Z        DEBUG   websocket/pool.go:119 (*incomerPool).AddClient  f31c687047b5: AddClient Test for pool /ws/publisher type : [*websocket.Pool]
2019-11-25T13:25:50.917Z        DEBUG   websocket/pool.go:91 (*Pool).OnDisconnected     f31c687047b5: OnDisconnected Test for pool /ws/publisher type : [*websocket.Pool]
2019-11-25T13:25:50.917Z        DEBUG   websocket/pool.go:145 (*Pool).RemoveClient      f31c687047b5: Successfully removed client Test for pool /ws/publisher type : [*websocket.Pool]
```

**Comment by safchain** [comment]
Addressed by : #2128

**Comment by safchain** [comment]
Closing it as it has been addressed, feel free to re-open if needed
